src/main/java/net/mcreator/ui/views/editor/image/tool/action/StampToolAction.java

```diff
@@ -19,12 +19,13 @@
 package net.mcreator.ui.views.editor.image.tool.action;
 
 import net.mcreator.ui.action.ActionRegistry;
+import net.mcreator.ui.init.L10N;
 import net.mcreator.ui.init.UIRES;
 import net.mcreator.ui.views.editor.image.tool.tools.StampTool;
 
 public class StampToolAction extends ToolChangeAction {
 	public StampToolAction(ActionRegistry actionRegistry) {
-		super(actionRegistry, "Stamp", "A tool for applying textures from templates", StampTool.class);
+		super(actionRegistry, L10N.t("dialog.imageeditor.stamp_tool_name"), L10N.t("dialog.imageeditor.stamp_tool_description"), StampTool.class);

```

Comment:
Where is this defined?

Reply:
?

Reply:
dialog.imageeditor.stamp_tool_name is not in texts, same for other one

Reply:
This is still missing in texts file